CHANGELOG.md

```diff
@@ -37,10 +37,12 @@
     - show
   - snowflake.snowpark.DataFrameWriter:
     - save_as_table
+- Added support for snow:// URLs to `snowflake.snowpark.Session.file.get` and `snowflake.snowpark.Session.file.get_stream`
```

Comment:
Is there support for PUT too?

Reply:
Technically snow:// URLs will be correctly parsed for PUT but writing to snow:// stages is limited